src/osemgrep/cli_ci/Meta.mli

```diff
@@ -0,0 +1,66 @@
+(** Collect information about the project from the environment. *)
```

Comment:
Can you rename to Project_metadata.ml?

And in the .ml toplevel comment says it was translated from meta.py